src/main/java/com/practicesoftwaretesting/apiTest/cart/CartController.java

```diff
@@ -1,12 +1,11 @@
-package com.practicesoftwaretesting.cart;
+package com.practicesoftwaretesting.apiTest.cart;
```

Comment:
do not use uppercase letters in packages. by the way `apiTests` looks not so good in `main`, why not to use just `api` and `ui` for packages